### PR TITLE
Add AccountLoader step to the login view

### PR DESCRIPTION
**Pull request by elias-garcia**
### What does this PR does?

This PR adds the second step "AccountLoader" to the login screen.

### How to test?

You should see a new step in the login screen when trying to login.

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Respect code style and lint
- [x] Update documentation (if needed)

### CODE REVIEW

File: src/views/login/components/account-loader/account-loader.view.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { FC } from "react";
+
+import useAccountLoaderStyles from "src/views/login/components/account-loader/account-loader.styles";
+import { WalletName } from "src/domain";
+import Typography from "src/views/shared/typography/typography.view";
+import WalletIcon from "../wallet-icon/wallet-icon.view";
+
+interface AccountLoaderProps {
+  selectedWallet?: WalletName;
```
Comment (amonsosanz) [inline src/views/login/components/account-loader/account-loader.view.tsx:9]: Why can this be undefined?

File: src/views/login/login.view.tsx [hunk 2]
```diff
@@ -36,49 +38,21 @@ const Login: FC = () => {
...
-                <Typography type="body1" className={classes.walletName}>
-                  WalletConnect
-                </Typography>
-                <Typography type="body2">Connect using mobile wallet</Typography>
-              </div>
-            </div>
-            <CaretRightIcon />
-          </li>
-        </ul>
+        {account.status === "pending" || account.status === "failed" ? (
```
Comment (amonsosanz) [inline src/views/login/login.view.tsx:41]: Here I'm missing `|| selectedWallet === undefined`

File: src/views/login/components/account-loader/account-loader.view.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+const AccountLoader: FC<AccountLoaderProps> = ({ selectedWallet }) => {
+  const classes = useAccountLoaderStyles();
+
+  switch (selectedWallet) {
```
[the remark this answers is not in the document]
Comment (elias-garcia) [reply src/views/login/components/account-loader/account-loader.view.tsx:15]: Nope, you also need the corresponding Icon.

File: src/views/login/login.view.tsx (hunk 2 above)
Comment (elias-garcia) [reply src/views/login/login.view.tsx]: The problem is that I wasn't resetting the `selectedWallet` to `undefined` in any case (which is wrong). I've uploaded a different approach. I think that the code looks cleaner and we depend only on the local state of the view to render it.

File: src/views/login/components/account-loader/account-loader.view.tsx (hunk 1 above)
Comment (elias-garcia) [reply src/views/login/components/account-loader/account-loader.view.tsx]: Right! Fixed.

File: src/views/login/components/account-loader/account-loader.view.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (adrifdez) [reply src/views/login/components/account-loader/account-loader.view.tsx:15]: You can pass `selectedWallet` to `WalletIcon`

File: src/views/login/login.view.tsx (hunk 2 above)
Comment (amonsosanz) [reply src/views/login/login.view.tsx]: Beautiful

**Review by amonsosanz — APPROVED** [review]

**Review by adrifdez — APPROVED** [review]

**Pull request merged**